### ISSUE TITLE
Add permission to read `/run/user/$id/at-spi/bus` by default

### ISSUE THREAD

**Issue by fiftydinar** [Samueru-sama/simple-appimage-sandbox#13]
That should fix this error in DeadBeeF:

```
(deadbeef:2): dbind-WARNING **: 07:29:55.145: Couldn't connect to accessibility bus: Failed to connect to socket /run/user/1000/at-spi/bus: File o directory non esistente
```

**Comment by fiftydinar** [comment]
Adding `--add-file /run/user/1000/at-spi/bus` fixes it manually.

**Comment by Samueru-sama** [comment]
How do flatpaks handle at-spi?

Also besides the warning does it actually break anything?

**Comment by Samueru-sama** [comment]
Seems they only allow a limited version at-spi and only if the app requests so. 

I don't think it is a good idea to allow this by default. 

I also need to update the script to only allow X11 access when on X11, else only allow the wayland socket since the X11 socket is a sandbox escape vector.

**Comment by fiftydinar** [comment]
> I also need to update the script to only allow X11 access when on X11, else only allow the wayland socket since the X11 socket is a sandbox escape vector.

Will this work for apps that are XWayland on Wayland?

**Comment by Samueru-sama** [comment]
> > I also need to update the script to only allow X11 access when on X11, else only allow the wayland socket since the X11 socket is a sandbox escape vector.
> 
> Will this work for apps that are XWayland on Wayland?

I don't think so, in those cases you will have to explicitly pass `--add-socket X11` flag
